app/src/main/res/layout/activity_component_detail.xml

```diff
@@ -25,10 +25,9 @@
 
   </com.google.android.material.appbar.AppBarLayout>
 
-  <net.skyscanner.backpack.demo.components.LockableNestedScrollView
+  <FrameLayout
```

Comment:
When the CoordinatorLayout is inside scroll view it's scrolling behaviour breaks. Therefore I removed scrollable container behaviour for all the component details.